tartiflette_plugin_scalars/geo_json.py

```diff
@@ -2,11 +2,11 @@
 
 from typing import Any
 
-import geojson
-
 from tartiflette.constants import UNDEFINED_VALUE
 from tartiflette.language.ast import StringValueNode
 
+import geojson
```

Comment:
I think that `geojson` should be added to the `isort.cfg` file on the `known_first_party ` statement (`dateutil` too isn't?):
https://github.com/tartiflette/tartiflette-plugin-scalars/blob/788a3cc95d5d50538c1b62f7bfe94ca20cc6e9ac/.isort.cfg#L5